### DISCUSSION TITLE
Is this category seperator comming soon?

### DISCUSSION THREAD

**Discussion by Paraxdev** [mantinedev/mantine#123] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/29353794/123999487-dc75cf80-d9d2-11eb-998e-0e0e15946c1f.png)
In the docs there are these categories object and I look for something like this for ages!
Is there a term for this kind of separator / list? 
And also does this maybe come soon with a mantine release?

**Comment by Paraxdev** [comment]
Also, is there a dev release or something where I can test the indev features?
I would love to test and give feedback or report possible bugs

**Reply by rtivital** [reply]
Hi Dominik, sure, it would be great if you could test out new features, to do so:
- clone this repository
- switch to next-minor branch: `git checkout next-minor`
- install dependencies `yarn`
- run docs site locally `npm run docs`

You will find current alpha version in the header next to the logo, you can update your dependencies to this version in `package.json` file and try out new features.

Current alpha version is `1.4.0-alpha.1`, it includes new styles API feature, several new hooks, components and 2 new packages: @mantine/dates and @mantine/prism.

Feel free to open new discussions and give feedback about new features, thanks in advance!

**Comment by rtivital** [comment, marked as answer]
Regarding navigation component: I have plans to add it to the new @mantine/ui package but will not be done until version 3.0.0, my estimation is that it will be released in mid November – start of December. Until that release, feel free to copy component from Mantine docs website, you can [find it here](https://github.com/mantinedev/mantine/blob/next-minor/docs/src/components/Layout/Navbar/NavbarDocsCategory/NavbarDocsCategory.tsx)
